apps/meteor/client/components/Omnichannel/hooks/useDepartmentsList.ts

```diff
@@ -44,7 +43,7 @@ export const useDepartmentsList = (
 				count: end + start,
 				sort: `{ "name": 1 }`,
 				excludeDepartmentId: options.excludeDepartmentId,
-				enabled: options.enabled,
+				enabled: options.enabled ? 'true' : 'false',
```

Comment:
makes sense but I prefer being a little more verbose here, and maybe if somebody uses this function in a javascript code and pass a wrong value it will lead in to a problem